doc/OnlineDocs/related_packages.rst

```diff
@@ -0,0 +1,43 @@
+Related Packages
+================
+
+The following is list of software packages that utilize or build off
+of Pyomo. This is certainly not a comprehensive list. [#f1]_
```

Comment:
This list will (hopefully) grow to be quite large.  Does it make sense to start subdividing it into sections?  Maybe things like 
- modeling extensions
- solvers and solution strategies
- domain-specific applications

...and packages could appear multiple times.

Reply:
Great suggestion. I completely agree.